src/scripts/main.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
```

Comment:
Ensure that the `table` element exists before accessing its properties. If the table is not present in the DOM, `table.rows` will be undefined, leading to runtime errors.